library/core/src/unicode/unicode_data.rs

```diff
@@ -17,14 +18,14 @@ fn bitset_search<
     let bucket_idx = (needle / 64) as usize;
     let chunk_map_idx = bucket_idx / CHUNK_SIZE;
     let chunk_piece = bucket_idx % CHUNK_SIZE;
-    let chunk_idx = if let Some(&v) = chunk_idx_map.get(chunk_map_idx) {
-        v
+    let chunk_idx = if chunk_map_idx < chunk_idx_map.len() {
+        chunk_idx_map[chunk_map_idx]
```

Comment:
There and lower can be added FIXME with note, about that this can be changed back to `.get()` when/if it will be const too. PR should be marked with `const-hack` label?

Reply:
@rustbot label +const-hack